lib/audumbla/enrichments/coarse_geocode.rb

```diff
@@ -0,0 +1,203 @@
+  # data will be used as context for evaluating interpretations. For example: 
+  # a `Place` with an existing latitude and longitude will verify that the
+  # point is within the bounding box for a candidate match.
+  # 
+  # `skos:exactMatch` are reserved for the GeoNames features returned by the 
+  # geocoder. Other matching URIs (currently: LC authorities) are included as
+  # `skos:closeMatch`
+  #
+  # @example enriching from a `#providedLabel`
+  #   
+  #    place = DPLA::MAP::Place.new.tap { |p| p.providedLabel = 'Georgia' }
+  #    CoarseGeocode.new.enrich_value.dump :ttl
+  #    # [
+  #    #    a <http://www.europeana.eu/schemas/edm/Place>;
+  #    #    <http://dp.la/about/map/providedLabel> "Georgia";
+  #    #    <http://www.geonames.org/ontology#countryCode> "US";
+  #    #    <http://www.w3.org/2003/01/geo/wgs84_pos#lat> 3.275042e1;
+  #    #    <http://www.w3.org/2003/01/geo/wgs84_pos#long> -8.350018e1;
+  #    #    <http://www.w3.org/2004/02/skos/core#closeMatch> <http://id.loc.gov/authorities/names/n79023113>;
+  #    #    <http://www.w3.org/2004/02/skos/core#exactMatch> <http://sws.geonames.org/4197000/>;
+  #    #    <http://www.w3.org/2004/02/skos/core#prefLabel> "Georgia, United States"
+  #    # ] .
+  #
+  # @example enriching from a `#providedLabel` with lat/lng guidance
+  #   
+  #    place = DPLA::MAP::Place.new.tap do |p| 
+  #      p.providedLabel = 'Georgia'
+  #      p.lat = 41.9997
+  #      p.long = 43.4998
+  #    end
+  #    
+  #    CoarseGeocode.new.enrich_value.dump :ttl
+  #    # [
+  #    #    a <http://www.europeana.eu/schemas/edm/Place>;
+  #    #    <http://dp.la/about/map/providedLabel> "Georgia";
+  #    #    <http://www.geonames.org/ontology#countryCode> "GE";
+  #    #    <http://www.w3.org/2003/01/geo/wgs84_pos#lat> 4.199998e1;
+  #    #    <http://www.w3.org/2003/01/geo/wgs84_pos#long> 4.34999e1;
+  #    #    <http://www.w3.org/2004/02/skos/core#exactMatch> <http://sws.geonames.org/614540/>;
+  #    #    <http://www.w3.org/2004/02/skos/core#prefLabel> "Georgia"
+  #    # ] .
+  #
+  class CoarseGeocode
+    include Audumbla::FieldEnrichment
+
+    DEFAULT_DISTANCE_THRESHOLD_KMS = 100
+
+    Twofishes.configure do |config|
+      config.host = 'localhost' 
+      config.port = 8080
+      config.timeout = 100
+      config.retries = 2 
+    end
+
+    ##
+    # Enriches the given value against the TwoFishes coarse geocoder. This 
+    # process adds a `skos:exactMatch` for a matching GeoNames URI, if any, and
+    # populates the remaining place data to the degree possible from the matched
+    # feature.
+    # 
+    # @param [DPLA::MAP::Place] value  the place to geocode
+    #
+    # @return [DPLA::MAP::Place] the inital place, enriched via coarse geocoding
+    def enrich_value(value)
+      return value unless value.is_a? DPLA::MAP::Place
+      interpretations = geocode(value.providedLabel.first, 
+                                [],
+                                maxInterpretations: 5)
```

Comment:
It seemed, in ad-hoc testing, like the quality of matches dropped quickly after the third or fourth.  The goal here is to limit the request cost for Twofishes (and the response sizes) while still getting the interpretations we need.  5 seemed like a good starting value.

This is also configurable in the latest commit.